tsconfig.json

```
@@ -0,0 +1,12 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "ES5",
+    "declaration": true,
+    "allowSyntheticDefaultImports": true,
+    "outDir": "./lib",
+    "moduleResolution": "node",
+    "lib": ["ES2020", "DOM"]
```

Let's import the helpers. It requires that you add `tslib` to the dependencies but it allows for further JavaScript de-duplication.

```suggestion
    "lib": ["ES2020", "DOM"],
    "importHelpers": true
```